examples/CMakeLists.txt

```diff
@@ -76,13 +76,34 @@ add_test(
     && ${PYTHON_EXECUTABLE} ${JSON_COMPARE} bend.out.json out.json --tol 0.03"
         WORKING_DIRECTORY ${EXAMPLES_DIR}/bend)
 
+add_test(
+        NAME gromosBend
```

Comment:
gromosBend --> gromos-bend would make it more consistent with the other tests

Reply:
We use snake_case instead now. Fixed with ff815cc.